ocaml/xapi/create_misc.ml

```diff
@@ -387,7 +387,38 @@ let make_packs_info () =
 		in
 		Array.fold_left (fun l fname -> get_pack_details fname @ l) [] packs
 	with _ -> []
-  
+
+let trim s = String.filter_chars s (not ++ String.isspace)
+
+let make_xen_livepatch_list () = 
```

Comment:
It would be good to add a comment above here that explains what the format of the response looks like (and for the function below as well).